scan.go

```diff
@@ -167,6 +173,13 @@ func newScanCommand() *cobra.Command {
 	cmd.Flags().DurationVarP(&malwareAnalysisTimeout, "malware-analysis-timeout", "", 5*time.Minute,
 		"Timeout for malicious package analysis")
 
+	// mark code-db flag required only when dependencyUsageEvidence is enabled
+	cmd.PreRun = func(cmd *cobra.Command, args []string) {
```

Comment:
This is good. But if this function returns an `error` then `cobra` shows the entire help after printing the error. This is a confusing user experience because the user will have to scroll back to find the error. I think we can do something like this:

https://github.com/safedep/vet/pull/335/files

Reply:
yep this one is more readable